eqcorrscan/utils/plotting.py

```diff
@@ -1354,6 +1360,32 @@ def pretty_template_plot(template, background=False, picks=False, **kwargs):
     else:
         mintime = background.sort(['starttime'])[0].stats.starttime
     template.sort(['network', 'station', 'starttime'])
+    # Sort the template by epicentral distance
+    if event:
+        if event.picks:
```

Comment:
I had included the outermost check to guard against an error if no event is supplied to the plotting function - but when refactoring this is no problem to change it. 
One question: what do you think should the function return if it can't sort? The original order or `None`?

Reply:
Probably `None`, it would be strange to return the original order. Then here you can just catch `None` and do some other sort.

Reply:
thanks, now it returns `None´.